great_expectations/expectations/metrics/query_metrics/query_column.py

```diff
@@ -35,7 +35,7 @@ def _sqlalchemy(
         metric_value_kwargs: dict,
```

Comment:
This file had `7` typing errors

https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L100

Reply:
Taken care of.